Makefile

```diff
@@ -3,9 +3,14 @@
 # Credits to Paul Sokolovsky (@pfalcon) for esp-open-sdk
 # Credits to Fabien Poussin (@fpoussin) for xtensa-lx106-elf build script
 #
+# (@Juppit) adaptation for gcc-4.9.2 and Cygwin
+# (@Juppit) touch makedir after make 
 
 
-VENDOR_SDK_VERSION = 1.4.0
+# for utils you need the tools pyinstaller and mcs
+BUILD_UTILS = n
```

Comment:
Yea, of course, it is only my personally preference, which should not be 
there in general.

Am 23.12.2015 um 01:41 schrieb Dmitry Kireev:
> In Makefile
> <https://github.com/kireevco/esp-alt-sdk/pull/8#discussion_r48314180>:
>
>>
>>
>> -VENDOR_SDK_VERSION = 1.4.0
>> +# for utils you need the tools pyinstaller and mcs
>> +BUILD_UTILS = n
>
> I would prefer to have this enabled by default. you get mcs and
> pyinstaller when configuring environment via scripts from env/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kireevco/esp-alt-sdk/pull/8/files#r48314180>.
>